web/sites/default/config/core.base_field_override.node.alert.status.yml

```diff
@@ -11,7 +11,7 @@ bundle: alert
 label: Published
 description: ''
 required: false
-translatable: false
```

Comment:
@RichardDavies , since this is a select list text field, should it be translatable? I wouldn't see us using a difference status in Spanish for an an alert than in English. 🤔

Reply:
Ah shoot... I just realized this is the core status value and not field_alert_status. Nothing to see here. Carry on.